guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1109 @@
+
+Refer to the [Threads](/popular-topics/threads) section of this guide to start using Threads.
+
+## Voice
+
+Support for voice has been separated into its own module. You now need to install and use [@discordjs/voice](https://github.com/discordjs/voice) for interacting with the Discord Voice API.
+
+## Customizable Manager Caches
+
+A popular request that has finally been heard - the `Client` class supports a new option, `makeCache` which accepts a `CacheFactory`.
+
+By combining this with the helper function `Options.cacheWithLimits` users can define custom caps on the caches of each Manager, and let discord.js handle the rest.
+
+```js
+const client = new Client({
+	makeCache: Options.cacheWithLimits({
+		MessageManager: 200, // This is default.
+		PresenceManager: 0, // Add more class names here.
+	}),
+});
+```
+
+Additional flexibility can be gained by providing a function which returns a custom cache implementation - keep in mind this should still maitain the Collector / Map-like interface for internal compatibility.
+
+```js
+const client = new Client({
+	makeCache: manager => {
+		if (manager.name === 'MessageManager') return new LimitedCollection(0);
+		return new Collection();
+	},
+});
+```
+
+## Commonly used methods that changed
+
+### Sending Messages, MessageEmbeds, and everything else
+
+With the introduction of Interactions and it becoming far common for users to want to send an embed with MessageOptions, methods that send messages now enforce a single param. 
+
+That can be either a string, an `APIMessage`, or that method's variant of `MessageOptions`.
+
+Additionally, all messages sent by bots now support up to 10 embeds. As a result the `embed` option is completely removed, replaced with an `embeds` array which must be in the options object.
+
+```diff
+- channel.send(embed);
++ channel.send({ embeds: [embed, embed2] });
+
+- channel.send('Hello!', { embed });
++ channel.send({ content: 'Hello!', embeds: [embed, embed2] });
+
+- interaction.reply('Hello!', { ephemeral: true });
++ interaction.reply({ content: 'Hello!', ephemeral: true });
+```
+
+`MessageEmbed#attachFiles` has been removed - files should now be attached directly to the Message.
+
+```diff
+- const embed = new Discord.MessageEmbed().setTitle('Attachments').attachFiles(['./image1.png', './image2.jpg']);
+- channel.send(embed);
++ const embed = new Discord.MessageEmbed().setTitle('Attachments');
++ channel.send({ embeds: [embed], files: ['./image1.png', './image2.jpg'] });
+```
+
+The `code` and `split` options have also been removed. This functionality will now have to be handled manually, such as via the `Formatters.codeBlock` and `Util.splitMessage` helpers.
+
+### Strings
+
+Many methods in discord.js that were documented as accepting strings would accept other types, and resolve this into a string on your behalf. The results of this behaviour were often undesirable, producing output such as `[object Object]`.
```

Comment:
Not assuming, but since most pages in the guide use American English.
```suggestion
Many methods in discord.js that were documented as accepting strings would accept other types, and resolve this into a string on your behalf. The results of this behavior were often undesirable, producing output such as `[object Object]`.
```